src/components/nodes-table/ColumnFilterString.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+  <v-card-text>
+    <v-row>
+      <v-col>
+        <v-text-field
+          label="Match Regex"
```

Comment:
I would use `Search` and in the hint text describe that this could be a regex.

Reply:
Done.